src/Selenium2Library/keywords/_selectelement.py

```diff
@@ -8,15 +8,24 @@ class _SelectElementKeywords(KeywordGroup):
 
     # Public
 
-    def get_list_items(self, locator):
-        """Returns the values in the select list identified by `locator`.
+    def get_list_items(self, locator, label=True):
```

Comment:
It would be good to explain what the `label` argument actually does in the documentation.